packages/commerce-sdk-react/src/hooks/types.ts

```diff
@@ -192,6 +192,7 @@ export type CacheUpdate = {
     update?: CacheUpdateUpdate<unknown>[]
     invalidate?: CacheUpdateInvalidate[]
     remove?: CacheUpdateRemove[]
+    clear?: boolean
```

Comment:
This is a new cache update action. If this is set to true, the entire cache is cleared. It was necessary to do this here as there was no access to the queryClient elsewhere (like the updater function).